scripts/ntlm-methods.nse

```diff
@@ -0,0 +1,58 @@
+local http = require "http"
+local shortport = require "shortport"
+local stdnse = require "stdnse"
+local string = require "string"
+local table = require "table"
+
+description = [[
+Returns authentication methods a winrm server supports.
+]]
+
+---
+-- @usage
+-- nmap --script winrm-auth-methods <target>
```

Comment:
Since ports 5985 and 5986 are not part of the default 1000 Nmap ports (currently), this usage statement is incorrect. Either put in the `-p 5985` or leave out the usage statement (NSEdoc generates one for the web automatically that is basically what you already have).